Update VFS/VSP a11y processes

1Copenut opened the pull request:
* Adding information about the testing accessibility specialists will do
* Moved screen reader testing to recommended testing, with a note that teams should do these tests if possible

1Copenut committed:
Update Staging Review Processes document

* Adding information about the testing accessibility specialists will do
* Moved screen reader testing to recommended testing, with a note that teams should do these tests if possible

1Copenut committed:
Update accessibility.md

* Adding a number of clarifications for VFS team specialist responsibilities
* Adding some links to DEPO platform a11y docs
* TODO: Update links when. we merge these documents

shiragoodman commented on platform/working-with-vsp/roles-and-responsibilities/accessibility.md:
```diff
@@ -1,29 +1,87 @@
+
+These are the expected responsibilities and division of labor for teams with one or more accessibility specialists. This section will outline each collaboration cycle meeting.
+
+- **[Project kickoff](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#project-kickoff)**
```
Capitalize Kickoff

shiragoodman commented on platform/working-with-vsp/roles-and-responsibilities/accessibility.md:
```diff
@@ -1,29 +1,87 @@
+  - VFS accessibility specialists are encouraged to attend the project kickoff meeting. Project kickoffs are a good opportunity for teams to ask questions and learn about the new product or feature.
+  - DEPO platform will collect specialist feedback (if any) and provide it to the VFS product team
+
+- **[Design intent](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#design-intent-collaboration)**
```
Design Intent Collaboration*

shiragoodman commented on platform/working-with-vsp/roles-and-responsibilities/accessibility.md:
```diff
@@ -1,29 +1,87 @@
+  - VFS specialists will answer questions and consult on accessibility issues
+  - VFS specialists will work with DEPO platform if there is a conflict or further collaboration is needed
+
+- **[Usability testing prep](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#usability-testing-prep)**
```
Usability Testing Prep*

shiragoodman commented on platform/working-with-vsp/roles-and-responsibilities/accessibility.md:
```diff
@@ -1,29 +1,87 @@
+  - DEPO platform will deliver findings and notify the product team of any launch blocking issues
+  - VFS specialists will answer questions and help the product team resolve issues
+
+- **[Full accessibility and 508 office audit](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#full-accessibility-and-508-office-audit)**
```
Full Accessibility and 508 Office Audit*

1Copenut committed:
Updated capitalization of meetings

1Copenut committed:
Update staging-review-processes.md

1Copenut committed:
Update staging-review-processes.md

1Copenut committed:
Update staging-review-processes.md

Revised the language about screen reader testing recommendation

1Copenut committed:
Update post-launch-audit-processes.md

Revising the post-launch audit language to reflect latest VSP/VFS adjustments for scalability and workflow.

1Copenut committed:
Update post-launch-audit-processes.md

andreahewitt-odd commented on platform/accessibility/guidance/post-launch-audit-processes.md:
```diff
@@ -14,7 +14,10 @@ The post-launch audit will focus on manual testing with mobile and assistive tec
-* Issues will be triaged according to their severity. Some issues will be launch blockers. Others can be fixed post-launch. Teams should review the [accessibility defect severity rubric](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md) if they have questions about issue severity or impact.
-* The accessibility specialist will communciate the application or content's readiness for review to the VA 508 office. The specialist will list known issues and serve as a liaison between the requesting team and the VA 508 representative.
-* Accessibility specialists will work with stakeholders as issues are ready to be re-tested. When an issue has been resolved, the specialist or project manager should close it with a brief comment like _"Issue has been tested, and success criteria met. Moving to close."_
+* Issues will be collected as issues in a new ZenHub epic. The accessibility specialist will assign this epic to the project manager (PM) after the audit is complete.
```
I think you mean product manager here

andreahewitt-odd replied:
Actually you mention Project Manager many places shere I think you mean product

1Copenut replied:
Did a full text find and replace, got all of the instances changed to product managers.

andreahewitt-odd commented on platform/accessibility/guidance/staging-review-processes.md:
```diff
@@ -45,15 +45,22 @@ OR
+- MacOS + Safari + VoiceOver
+- iOS + Safari + Mobile VoiceOver (as time permits)
+
+The accessibility specialist attaches the tickets to a single ZenHub epic and assigns the epic to the person who requested the staging review. During the staging review meeting, you and your team can ask questions about any of the issues identified. Applications are not considered Section 508 compliant until all severity 0, 1, 2, and 3 issues are remediated. Severity 4 items include minor issues  ...
```
Are applications 508 compliant if they have severity 4 items? I was under the impression that they aren't but this reads like they are.

1Copenut replied:
Some of the 508 severity 4 items in the past were recommendations that didn't mean the app was out of compliance. I've added a `508-enhancement` tag to capture these things that don't rise to the level of defects.

emilywaggoner commented on platform/accessibility/guidance/staging-review-processes.md:
```diff
@@ -13,16 +13,16 @@ Staging reviews test the [happy path](https://en.wikipedia.org/wiki/Happy_path)
 - [ ] Keyboard navigation - [read more about navigating with the keyboard](#keyboard-navigation-check)
-- [ ] Screen reader tests (VoiceOver + Safari or NVDA + Firefox) - [read more about getting started with screen readers](#screen-reader-tests)
 
 ## Recommended accessibility tests
```
Rachael mentioned calling these "Advanced accessibility tests" and the previous section "Baseline" - Just putting that there for you to consider, not a required change I don't think.

1Copenut replied:
I like that. I fudged a little bit and used the term "foundational", but I've been baking that into conversations so feel it works here too.

1Copenut replied:
👍

emilywaggoner commented on platform/working-with-vsp/roles-and-responsibilities/accessibility.md:
```diff
@@ -1,29 +1,87 @@
+- **[Project Kickoff](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#project-kickoff)**
+
+  - VFS accessibility specialists are encouraged to attend the project kickoff meeting. Project kickoffs are a good opportunity for teams to ask questions and learn about the new product or feature.
+  - DEPO platform will collect specialist feedback (if any) and provide it to the VFS product team
```
What do you think about saying "...collect and review specialist feedback..."?

1Copenut replied:
👍

emilywaggoner commented on platform/working-with-vsp/roles-and-responsibilities/accessibility.md:
```diff
@@ -1,29 +1,87 @@
+- **[Design Intent](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#design-intent-collaboration)**
+
+  - VFS accessibility specialists are encouraged to attend the design intent meeting
+  - DEPO platform will collect specialist feedback and provide it to the VFS product team. DEPO platform will comment or add guidance as needed.
```
Same thought as above - "...collect and review specialist feedback..."

1Copenut replied:
👍

emilywaggoner reviewed (approved):
Left a couple of minor notes throughout, otherwise I think it looks good. Thanks!

1Copenut committed:
Changing project to program

Was using the term "project manager" when I needed to use "program manager".

1Copenut committed:
Changing project to product

Was using the wrong term for product managers, so updating for consistency and accuracy

1Copenut committed:
Simplifying a data point

Making a simpler explanation under the project kickoff

1Copenut committed:
Simplifying a data point

Updating a sentence to use simpler language for collecting feedback.

1Copenut committed:
Changing language for testing

Using the same language we've been rolling out for foundational accessibility testing and advanced accessibility testing.

1Copenut committed:
Updated staging review processes

* Added the distinctions for VFS teams with and without an accessibility specialist
* Re-nesting the headings for foundational testing
* Reordered content for better flow

1Copenut committed:
Update staging-review-processes.md

Fixing two internal links

1Copenut committed:
Update staging-review-processes.md

* Further reorganization and fixing a few links.

1Copenut committed:
Update post-launch-audit-processes.md

* Refactoring to include the VFS / DEPO Platform responsibilities

1Copenut committed:
Update post-launch-audit-processes.md

* Downcasing the headings

1Copenut committed:
Revising the R&R layout

* Removed the collaboration cycle breakouts
* Set TODO reminders for adding links to the staging and post-launch VFS guidance for a11y testing
* Added two bullet points for VFS team responsibilities

rroueche commented on platform/accessibility/guidance/post-launch-audit-processes.md:
```diff
@@ -13,8 +13,58 @@ The post-launch audit will focus on manual testing with mobile and assistive tec
+- [ ] Axe-coconut scans in daily development - [read more about the axe-coconut experimental plugin](https://www.deque.com/blog/test-leading-edge-accessibility-axe-coconut-axe-core-3-0/)
+- [ ] WAVE tool spot checks - [read more about the WebAIM WAVE tool](https://wave.webaim.org/)
+
+## What to expect from accessibility specialists during post-launch audit audit
```
Just a typo - you have "audit" written twice

1Copenut committed:
Removing a duplicated word
